portal-impl/src/com/liferay/portal/events/StartupAction.java

```diff
@@ -67,28 +62,6 @@ public void run(String[] ids) throws ActionException {
 
 	protected void doRun(String[] ids) throws Exception {
 
-		// Check Tomcat's lib/ext directory
```

Comment:
@julschong @tinatian This block of code was just added last month, you can not just delete it. This complain must be preserverd, Brian asked for it. You can remove the property, but still need to calculate out the location in order to scan for complain. Please fix and resend.

Reply:
@tinatian Hi Tina, I will inline _getLibDir into `StartupAction` and still remove the globalSharedLib prop